ncm-metaconfig/src/main/metaconfig/nrpe/pan/schema.pan

```diff
@@ -0,0 +1,27 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+declaration template metaconfig/nrpe/schema;
+
+include 'pan/types';
+
+@{Configuration options for Nagios Remote Plugin Executor (NRPE)}
+type nrpe_service = {
+    'log_facility'              : string = 'daemon'
+    'pid_file'                  : string = '/var/run/nrpe.pid'
```

Comment:
Presumably this **must** be an absolute path to a file, perhaps we should split out the inner part of `valid_absolute_file_paths` to allow it to be used on a single string.